catalog/pipeline/fbc-release/0.11/README.md

```diff
@@ -0,0 +1,74 @@
+# FBC Release Pipeline
+
+ Tekton release pipeline to interact with FBC Pipeline                       
+
+## Parameters
```

Comment:
you added parameters to the pipeline definition and didn't change the parameters in the readme at all

Reply:
updated.